pegasus/test/fixtures/deliverer/expected/hoc_signup_2023_receipt_en/body.html

```diff
@@ -22,21 +22,21 @@ <h3>3. Invite a local volunteer to inspire your students</h3>
+<p>Swag is a great way to get students excited about the Hour of Code and reward them for completing their activity. At the <a href="https://test-store.code.org">Code.org store</a>, you can order Hour of Code swag kits, fun stickers, buttons, temporary tattoos, and more. But hurry, supplies are limited.</p>
 
 <h3>5. Encourage kids to continue learning</h3>
 
-<p>An Hour of Code is just the beginning! We hear over and over again how much students love the Hour of Code and discover a newfound interest in computer science. Encourage them to continue learning. Whether you're an administrator, teacher, or advocate, we have professional learning, free classroom curriculum and lesson plans, and resources to help you <a href="https://test.code.org/teach">bring computer science classes to your school</a> or expand your offerings.</p>
+<p>An Hour of Code is just the beginning! We hear over and over again how much students love the Hour of Code and discover a newfound interest in computer science. Encourage them to continue learning. Whether you're an administrator, teacher, or advocate, we have professional learning, free classroom curriculum and lesson plans, and resources to help you <a href="https://test.code.org/teach">bring computer science classes to your school</a> or expand your offerings.  </p>
 
-<p>Many of the organizations offering activities on HourofCode.com also have curriculum and professional learning available as well. We've highlighted <a href="https://test.hourofcode.com/beyond">curriculum providers that can help you or your students go beyond an hour.</a> </p>
+<p>Many of the organizations offering activities on HourofCode.com also have curriculum and professional learning available as well. We've highlighted <a href="https://test.hourofcode.com/beyond">curriculum providers that can help you or your students go beyond an hour.</a></p>
 
 <p>Hadi Partovi<br>
 Founder, Code.org<br></p>
 
 <hr>
 
 <p><small>
-You're receiving this email because you signed up for the Hour of Code, supported by more than 200 partners and organized by Code.org. Code.org is a 501c3 non-profit. Our address is <a href="https://www.google.com/maps/place/801+5th+Ave,+Seattle,+WA+98104/@47.6051193,-122.3336564,17z">801 5th Avenue, Suite 2100, Seattle, WA 98104</a>. Don't want these emails? <a href="">Unsubscribe</a>.
+You're receiving this email because you signed up for Hour of Code, supported by more than 200 partners and organized by Code.org. Code.org is a 501c3 non-profit. Our address is <a href="https://www.google.com/maps/place/801+5th+Ave,+Seattle,+WA+98104/@47.6051193,-122.3336564,17z">801 5th Avenue, Suite 2100, Seattle, WA 98104</a>. Don't want these emails? <a href="">Unsubscribe</a>.
```

Comment:
I just read on hourofcode.com that it says over 400 partners. If we can update without an issue great. Otherwise no worries

Reply:
On it!